### PR TITLE
Fixed Telegram notifier

### PR DESCRIPTION
**Pull request by toshka**
Saw that in #22 notifier raises Exception.

### CODE REVIEW

File: torrt/notifiers/telegram.py [hunk 1]
```diff
@@ -32,11 +32,13 @@ def test_configuration(self):
 
     def send_message(self, msg):
         url = '%s%s/sendMessage' % (self.url, self.token)
-        r = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
-        if r.json()['ok']:
-            LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
-        else:
-            LOGGER.error('Telegram notification not send: %s' % r['description'])
+        response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        if response.ok:
```
Comment (idlesign) [inline torrt/notifiers/telegram.py]: Wouldn't it better to raise an exception (so to have a chance to further investigate cause of an issue) with ``raise_for_status``?

File: torrt/notifiers/telegram.py [hunk 2]
```diff
@@ -32,13 +33,19 @@ def test_configuration(self):
...
-                LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
+        try:
+            response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        except RequestException as e:
+            LOGGER.error('Failed to send Telegram message: %s' % e)
+        else:
+            if response.ok:
+                json_data = response.json()
+                if json_data['ok']:
+                    LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
```
Comment (idlesign) [inline torrt/notifiers/telegram.py]: ``.debug()`` is probably sufficient not to clutter the log.

File: torrt/notifiers/telegram.py [hunk 3]
```diff
@@ -32,13 +33,19 @@ def test_configuration(self):
...
         url = '%s%s/sendMessage' % (self.url, self.token)
-        response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
-        if response.ok:
-            json_data = response.json()
-            if json_data['ok']:
-                LOGGER.info('Telegram message was sent to user %s' % self.chat_id)
+        try:
+            response = requests.post(url, data={'chat_id': self.chat_id, 'text': msg})
+        except RequestException as e:
+            LOGGER.error('Failed to send Telegram message: %s' % e)
```
Comment (idlesign) [inline torrt/notifiers/telegram.py]: There's no need to format (``%``) messages beforehand, just pass arguments.

Comment (toshka) [reply torrt/notifiers/telegram.py]: Wow! Thank you! All my life I used format for logging :(

File: torrt/notifiers/telegram.py (hunk 2 above)
Comment (toshka) [reply torrt/notifiers/telegram.py]: I'm thinking on it too. But INFO level used from initial commit. So we must add this to release notes.

Comment (idlesign) [reply torrt/notifiers/telegram.py]: Oh, I see. Let's have it as is.

**Comment by idlesign** [comment]
Thank you. Merged.

**Pull request merged**